lib/ts/Arena.h

```diff
@@ -21,12 +21,13 @@
   limitations under the License.
  */
 
-#ifndef __ARENA_H__
-#define __ARENA_H__
+#pragma once
```

Comment:
-1 on pragma in this PR. If we do pragma, raise it on `dev@`, get consensus and apply it to the whole codebase at once. This is not the right PR for this change.